csharp/global.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "sdk": {
+    "version": "1.0.0"
```

Comment:
not sure if we need the global.json. the problem with it is that it's only applied if a corresponding version of dotnet SDK exists, otherwise it's just ignored. So in a way it brings more confusion than what it's good for.